src/components/BlogPost.js

```diff
@@ -23,20 +20,21 @@ const ImageWrapper = styled.div`
 const BlogPost = ({ post }) => (
   <BlogPostWrapper>
     <Date date={post.publishedDate} />
-    {post.tags.map((tag) => (
-      <Tag tag={tag} />
+    {post.tags.map((tag, index) => (
+      <Tag key={index} tag={tag} />
```

Comment:
@dimanyc So I understand why I need keys and that they should be unique, but my tags don't have an ID of their own.  A few people suggested that I use an index, but then I found [this article](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318), so I'm thinking this isn't the best option.  I do plan to sort posts based on tags in the future, but I'm not sure if that is enough of a reason to avoid using an index...  I looked at using a [shortid](https://www.npmjs.com/package/shortid), but since my tags are created in Contentful, I don't see a way to generate them upon creation, and generating them in the component would mean they would change each page render, right? Do you have any suggestions on this front?